modules/cnf-topology-aware-lifecycle-operator-platform-update.adoc

```diff
@@ -0,0 +1,247 @@
+// Module included in the following assemblies:
+// Epic CNF-2600 (CNF-2133) (4.10), Story TELCODOCS-285
+// * scalability_and_performance/cnf-topology-aware-lifecycle-operator.adoc
+
+:_content-type: PROCEDURE
+[id="talo-platform-update_{context}"]
+= Performing a platform update
+
+.Prerequisites
+
+* Install the Topology Aware Lifecycle Operator.
+* Update ZTP to the latest version. 
+* Provision one or more managed clusters using ZTP.
+* Log in as a user with `cluster-admin` privileges.
+* Create {rh-rhacm} policies in the Hub cluster.
+
+For more information about how to update ZTP, see link:https://github.com/openshift-kni/cnf-features-deploy/blob/master/ztp/gitops-subscriptions/argocd/Upgrade.md[Upgrading GitOps ZTP].
+// This link^ has to be updated for OCP docs when ready.
+
+.Procedure
+
+. List the nodes and their cluster versions by running the following command:
++
+[source,terminal]
+----
+$ oc get clusterversion,nodes
+----
++
+.Example output
+
+[source,terminal]
+----
+NAME                                         VERSION   AVAILABLE   PROGRESSING   SINCE   STATUS
+clusterversion.config.openshift.io/version   4.9.15    True        False         4d16h   Cluster version is 4.9.15
+
+NAME                                           STATUS   ROLES           AGE    VERSION
+node/eko5.cnf20.example.redhat.com   Ready    master,worker   123d   v1.22.3+e790d7f
+node/eko6.cnf20.example.redhat.com   Ready    master,worker   123d   v1.22.3+e790d7f
+node/eko7.cnf20.example.redhat.com   Ready    master,worker   123d   v1.22.3+e790d7f
+----
+
+. Save the contents of your `PolicyGenTemplate` in the `du-upgrade.yaml` file.
+//This step has to be updated. Disconnected vs connected env 
++
+.Example of `PolicyGenTemplate` for platform update
++
+[source,yaml]
+----
+apiVersion: ran.openshift.io/v1
+kind: PolicyGenTemplate
+metadata:
+ name: "du-upgrade"
+ namespace: "ztp-group-du-sno"
+spec:
+ bindingRules:
+   group-du-sno: "" <1>
+ mcp: "master"
+ remediationAction: inform
+ sourceFiles:
+   - fileName: ClusterVersion.yaml
+     policyName: "platform-upgrade"
+     metadata:
+       name: version
+     spec:
+       channel: "stable-4.10"
```

Comment:
apart from the channel we do also have an upstream field that is configurable.. currently it points to the default upgrade-graph for openshift hosted here: https://api.openshift.com/api/upgrades_info/v1/graph
The openshift docs for clusterVersion
https://docs.okd.io/latest/rest_api/config_apis/clusterversion-config-openshift-io-v1.html

Reply:
@nishant-parekh I added this to the example file. Please, check the rest of the procedure as well.